lib/models/payment-processor-adapters/coinpayments.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const coinpayments = require('../../vendor/coinpayments');
+const constants = require('../../constants');
+
+let coinpaymentsProcessor;
+
+const coinpaymentsAdapter = {
+
+  serializeData: function (data) {
+    return data;
+  },
+
+  parseData: function () {
+    return coinpaymentsProcessor.rawData;
+  },
+
+  addPaymentMethod: function (token) {
+    return new Promise((resolve, reject) => {
+      coinpayments.getCallbackAddress(token, function(err, data) {
+        if (err) reject(err);
+        console.log('### addPaymentMethod adapter data: ', coinpaymentsProcessor.data);
```

Comment:
can we get access to the logger and use it instead of `console.log`? If we're going to log stuff I feel like we should do that.

Reply:
doesn't look like service storage models has a logger, but you can probably just copy bridge's: https://github.com/Storj/bridge/blob/master/lib/logger.js